snap/types.go

```diff
@@ -198,11 +198,13 @@ type ComponentType string
 const (
 	// TestComponent is just for testing purposes.
 	TestComponent ComponentType = "test"
+	// StandardComponent is for vanilla components with no special behavior.
+	StandardComponent ComponentType = "standard"
```

Comment:
Maybe worth getting id of the `test` type, since they are functionally the same?

Reply:
We want to do that eventually, but not immediately as we have some snaps that we use in spread tests and have this type and that we uploaded to the store. We need to change those first.